packages/ubuntu_wizard/lib/src/utils/window.dart

```diff
@@ -43,3 +43,6 @@ Future<void> setWindowTitle(String title) {
 Future<void> setWindowClosable(bool closable) {
   return windowManager.setClosable(closable);
 }
+
+/// Ensures initialization of the plugin(s) related to managing windows.
+Future<void> ensureInitialized() => windowManager.ensureInitialized();
```

Comment:
can we have both ensure initialized calls here?
```suggestion
Future<void> ensureInitialized() {
  WidgetsFlutterBinding.ensureInitialized();
  return windowManager.ensureInitialized();
}
```

Reply:
I swear I considered that idea and even calling the function something with "framework", but I got unsure whether I could be mixing responsibilities. I agree it looks nicer.